bindings/src/Capgemini.PowerApps.SpecFlowBindings/Configuration/TestConfiguration.cs

```diff
@@ -67,14 +92,29 @@ public Uri GetTestUrl()
         /// <returns>The user configuration.</returns>
         public UserConfiguration GetUser(string userAlias)
```

Comment:
It has just occurred to me that this may technically constitute a breaking change for some package consumers. We'll probably want to add an optional parameter e.g.

```suggestion
        public UserConfiguration GetUser(string userAlias, bool currentUser = true)
```

Based on `currentUser`, we can either iterate or return the user for current thread. Will need a new [ThreadStatic] property that we set via `aliasEnumerator.Current`. If `currentUser` is `true` we return the value in this property if set. If not, we `MoveNext` and provide it with the new user.

Reply:
Also need to update the XML doc comment for `currentUser`